go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/rh-ecosystem-edge/preinstall-utils
 go 1.20
 
 require (
+	github.com/go-openapi/swag v0.23.0
```

Comment:
We shouldn't have an entire OpenAPI swagger dependency just for swags just for `ContainsString`. If you need a contains function, either bring in the modern `lo` library made for this purpose or use the existing outdated (although already existing) `funk` dependency